<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraWidget</class>
 <widget class="QGroupBox" name="CameraWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>405</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="title">
   <string>Camera Settings</string>
  </property>
  <property name="checkable">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="projectionLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Projection Mode </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Perspective</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="horizontalFovLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Horiontal FOV </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="horizontalFovSpinBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="suffix">
         <string> Â°</string>
        </property>
        <property name="minimum">
         <double>30.000000000000000</double>
        </property>
        <property name="maximum">
         <double>120.000000000000000</double>
        </property>
        <property name="value">
         <double>45.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="sensorWidthLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Sensor Width </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="sensorWidthSpinBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="suffix">
         <string> mm</string>
        </property>
        <property name="minimum">
         <double>10.000000000000000</double>
        </property>
        <property name="maximum">
         <double>800.000000000000000</double>
        </property>
        <property name="value">
         <double>50.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="focalLengthLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Focal Length </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="focalLengthSpinBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="suffix">
         <string> mm</string>
        </property>
        <property name="minimum">
         <double>10.000000000000000</double>
        </property>
        <property name="maximum">
         <double>800.000000000000000</double>
        </property>
        <property name="value">
         <double>50.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="apertureLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Aperture </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="apertureSpinBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>22.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>2.800000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="shutterSpeedLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Shutter Speed </string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="shutterSpeedSpinBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="prefix">
         <string>1/</string>
        </property>
        <property name="suffix">
         <string> s</string>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>8000.000000000000000</double>
        </property>
        <property name="value">
         <double>60.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="focusDistanceLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Focus Distance </string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="focusDistanceSpinBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="minimum">
         <double>0.300000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>6.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="nearPlaneLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Near Plane </string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="nearPlaneSpinBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="farPlaneLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Far Plane </string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="farPlaneSpinBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>500.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="updateViewportCameraButton">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Update Viewport Camera from Render Camera</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="updateRenderCameraButton">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Update Render Camera from Viewport Camera</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
